### ISSUE TITLE
Documentation incomplete

### ISSUE THREAD

**Issue by andre-merzky** [saga-project/BigJob#124]
So, I am trying to find out how I can list running / known compute units.  Ah, nice -- the BigJob home page links to the API documentation.  Looking at that documentation at (_) shows *no_ methods on the `ComputeDataService` though (which is where the `CU`s get submitted -- for example  according to (*_) ) .  In fact, the `ComputeDataService` does not have any methods at all, and other `XUnitServices` are not listed either.  In fact, that document does not seem to list *any_ method for _any_ object (only attributes)??

So what, I'm cool - I go to the code, find the pilot/api subtree -  but there I see only an `__init__` method on the `ComputeDataService` (which, btw, takes one argument: `wds_id` with no explaination on what `WDS` means).  

Oh well, that class inherits from `compute.api.ComputeUnitService` and `data.api.PilotDataService`. The `ComputeUnitService` actually has a `submit_compute_unit` -- so this might be the one which is used in the example -- although its documentation (and the documentation in for all other classes / methods in that file) talk about work units etc.  But, there is no hint on how to list submitted units either.

But this is just the API, so I go to the implementation folder, have to grep through the files to find where the `ComputeUnitService` is implemented (files are named differently and organized differently than in the API tree), and then in `pilot_manager.py` I find:

```
class ComputeUnitService(object):
    def __init__(self):
        raise NotImplementedError("Please use ComputeDataService.")
```

Oh well, rinse and repeat, and I see two implementations.  Picking the decentral one (why not), I find that there is a `list_data_units`, but no `list_compute_units`.  WUT?

So, long story short: I am now very frustrated and still don't know how the hell I can list compute units?  And really, what is it with BigJob and Documentation??  Does it exist somewhere else?

(_) http://saga-project.github.io/BigJob/sphinxdoc/library/index.html
(_*) http://saga-project.github.io/BigJob/sphinxdoc/patterns/simple.html
